terraform/aws/projects/dandi.tfvars

```diff
@@ -8,19 +8,18 @@ user_buckets = {
   "scratch-staging" : {
     "delete_after" : 7
   },
-  "scratch" : {
+  "scratch-prod" : {
```

Comment:
I think we should keep these just as `scratch`, and not let the word `prod` leak into them.